tsrc/cli/init.py

```diff
@@ -1,17 +1,35 @@
-""" Entry point for `tsrc init` """
+""" tsrc init """
+
 import os
 
+import click
 import path
 import ui
 
+import tsrc.cli
 import tsrc.workspace
 
 
-def main(args):
-    workspace_path = args.workspace_path or os.getcwd()
+@click.command("init")
+@click.argument("url", nargs=1)
+@click.option("-b", "--branch", metavar="<branch>", is_flag=False, default="master")
+@click.option("-g", "--group", "groups", metavar="<group>", multiple=True)
+@click.option("-s", "--shallow", is_flag=True)
+@click.option("-w", "--workspace", metavar="<workspace>", is_flag=False)
```

Comment:
Why not using workspace_cli here. I bet init is spevial but I cant spot the difference.

Wont the main_wrapper be missing here?

Reply:
Is is phylosophical?

Reply:
Yes, we need `main_wrapper`

```
Why not using workspace_cli
```

Because the whole purpose of `workspace_cli` is to get an existing workspace.

Maybe we should name it `requires_workspace` or something